test/integration/targets/vmware_guest/tasks/template_cdrom_d1_c1_f0.yml

```diff
@@ -0,0 +1,113 @@
+- name: Wait for Flask controller to come up online
```

Comment:
Could you please add this test case [over here](https://github.com/ansible/ansible/blob/f3a3f424902d208c8fdbfa9983559251faad875a/test/integration/targets/vmware_guest/tasks/main.yml#L29) ? Comment this testcase as VCSIM does not support template operations.

Reply:
Yes, I commented out this test case in main.yml. Thanks.